test/functional/pdo_sqlsrv/pdo_574_next_rowset.phpt

```diff
@@ -0,0 +1,154 @@
+--TEST--
+GitHub issue 574 - Fetch Next Result Test
+--DESCRIPTION--
+Verifies the functionality of PDOStatement nextRowset
+--ENV--
+PHPT_EXEC=true
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+
+require_once("MsCommon_mid-refactor.inc");
+
+try {
+    $conn = connect();
+    $tableName = 'test574';
+    $tableName1 = 'test574_1';
+
+    // create two tables with max fields
+    $columns = array(new ColumnMeta('varchar(max)', 'col1'));
+    createTable($conn, $tableName, $columns);
+
+    $columns = array(new ColumnMeta('varchar(max)', 'col1'));
+    createTable($conn, $tableName1, $columns);
+
+    // insert one row to each table
+    $phrase = str_repeat('This is a test ', 250);
```

Comment:
if you're going to change the initial buffer size, this number will have to change as well